packages/oui-tooltip/_variables.less

```diff
@@ -1,6 +1,13 @@
 @oui-tooltip-min-width: rem-calc(100);
 @oui-tooltip-max-width: rem-calc(300);
-@oui-tooltip-margin: rem-calc(6) 0;
+@oui-tooltip-margin: rem-calc(10) 0;
+@oui-tooltip-margin_small: rem-calc(5) 0;
+@oui-tooltip-padding: rem-calc(10) rem-calc(20);
+@oui-tooltip-font-size: rem-calc(14);
+@oui-tooltip-font-color: @oui-color-sapphire;
+@oui-tooltip-box-shadow: 0 rem-calc(9) rem-calc(10) 0 fade(@oui-color-info-fade, 50%);
```

Comment:
Sadly `fade(@oui-color-info-fade, 50%)` is not the same than `#DFEAF8` (see ui specs). The last one is a bit darker on my side, what you chosen is almost invisible on my screen.

Reply:
I know. I do it this way because over others colors than white, it's ugly.